xbmc/platform/win10/filesystem/WinLibraryDirectory.cpp

```diff
@@ -18,14 +18,14 @@
  *
  */
 
-#include "WinLibraryDirectory.h"
```

Comment:
correct include order is, own header file first

Reply:
look like it's caused by `.clang-format` rules